src/test/java/com/ui_init_setup/practiceproject/pages/ParaBankAccountOverviewPage.java

```diff
@@ -0,0 +1,52 @@
+package com.ui_init_setup.practiceproject.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.LoadableComponent;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ParaBankAccountOverviewPage extends LoadableComponent<ParaBankAccountOverviewPage> {
+
+    private final WebDriver driver;
+    private final WebDriverWait wdWait;
+    public static final String PARA_BANK_ACCOUNT_OVERVIEW_URL = "https://parabank.parasoft.com/parabank/overview.htm";
```

Comment:
do you intend to use this variable outside of this class?

Reply:
no intentions to use it outside of this class